extensions/amp-story-subscriptions/validator-amp-story-subscriptions.protoascii

```diff
@@ -15,4 +15,21 @@ tags: {  # <amp-story-subscriptions>
   amp_layout: {
     supported_layouts: CONTAINER
   }
+  attrs: {
```

Comment:
You can sync with main branch now since the validator PR is merged: https://github.com/ampproject/amphtml/pull/38194